hack/common.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+GO111MODULE="on"
+GOPROXY="https://proxy.golang.org"
+GOROOT="${GOROOT:-go env GOROOT}"
+
+OUTDIR="build/_output"
+OUTDIR_BIN="build/_output/bin"
+OUTDIR_TEMPLATES="$OUTDIR/csv-templates"
+OUTDIR_CRDS="$OUTDIR_TEMPLATES/crds"
+OUTDIR_BUNDLEMANIFESTS="$OUTDIR_TEMPLATES/bundlemanifests"
+OUTDIR_TOOLS="$OUTDIR/tools"
+OUTDIR_CLUSTER_DEPLOY_MANIFESTS="$OUTDIR/cluster-deploy-manifests"
+
+# This env var allows developers to point to a custom oc tool that isn't in $PATH
+# defaults to just using the 'oc' binary provided in $PATH
+OC_PATH="${OC_PATH:-oc}"
+
```

Comment:
From line 3 to line 17, only line 8 appears to be required to run the functional tests. Can we delete the rest?

Reply:
Yes, that is an extra code. I was not sure if I need it for CI. I will remove it